docs/tools_scripts.md

```diff
@@ -258,3 +258,57 @@ python tools/deploy_test.py \
+    --workspace-size INT \
+    --fp16 \
+    --show \
+    --verify \
+    --verbose
+```
+
+Description of arguments:
+
+- `config` : The path of a model config file.
+- `model_type` :The model type of the config file, options: `inpainting`, `mattor`, `restorer`, `synthesizer`.
+- `img_path` : The path to input image file.
+- `onnx_file` : The path to input ONNX file.
+- `--trt-file` : The path of output TensorRT model. If not specified, it will be set to `tmp.trt`.
+- `--max-shape` : Maximum shape of model input.
+- `--min-shape` : Minimum shape of model input.
+- `--workspace-size`: Max workspace size in GiB. If not specified, it will be set to 1 GiB.
+- `--fp16`: Determines whether to export TensorRT with fp16 mode. If not specified, it will be set to `False`.
+- `--show`: Determines whether to show the output of ONNX and TensorRT. If not specified, it will be set to `False`.
+- `--verify`: Determines whether to verify the correctness of an exported model. If not specified, it will be set to `False`.
+- `--verbose`: Determines whether to verbose logging messages while creating TensorRT engine. If not specified, it will be set to `False`.
+
+**Note**: This tool is still experimental. Some customized operators are not supported for now. We only support `restorer` for now. While generating ONNX file of SRCNN, replace 'bicubic' with 'bilinear' in 40th line of mmedit/models/backbones/sr_backbones/srcnn.py. For TensorRT does not support bicubic interpolation by now and final performace will be weaken by about 4%.
+
+#### List of supported models exportable to TensorRT
+
+The table below lists the models that are guaranteed to be exportable to TensorRT engine and runnable in TensorRT.
+
+|  Model   |                                                                               Config                                                                                | Dynamic Shape | Batch Inference | Note  |
+| :------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----------: | :-------------: | :---: |
+|  ESRGAN  |       [esrgan_x4c64b23g32_g1_400k_div2k.py](https://github.com/open-mmlab/mmediting/blob/master/configs/restorers/esrgan/esrgan_x4c64b23g32_g1_400k_div2k.py)       |       Y       |        Y        |       |
+|  ESRGAN  | [esrgan_psnr_x4c64b23g32_g1_1000k_div2k.py](https://github.com/open-mmlab/mmediting/blob/master/configs/restorers/esrgan/esrgan_psnr_x4c64b23g32_g1_1000k_div2k.py) |       Y       |        Y        |       |
+|  SRCNN   |            [srcnn_x4k915_g1_1000k_div2k.py](https://github.com/open-mmlab/mmediting/blob/master/configs/restorers/srcnn/srcnn_x4k915_g1_1000k_div2k.py)             |       Y       |        Y        |  'bicubic' upsampling must be replaced with 'bilinear'     |
+
+**Notes**:
+
+- *All models above are tested with Pytorch==1.8.1,  onnxruntime==1.7.0 and tensorrt==7.2.3.4*
```

Comment:
Could we test with Pytorch==1.6.0 as well? Since 1.8.1 is relatively new to most users.